src/Misc/fileTools.ts

```diff
@@ -87,7 +87,11 @@ export class FileTools {
      * Gets or sets a function used to pre-process url before using them to load assets
      */
     public static PreprocessUrl = (url: string) => {
-        return url;
+        let processedURL = url.substring(5).toLowerCase();

```

Comment:
This can break back compat and the default behaviour can not be change for all the urls ???

Reply:
I agree, I don't think we want to do remove the "file:" prefix here, but just remove "./" from the front of our url

Reply:
Ah, I wasn't aware that PreprocessUrl was a reserved function, we can also make another function "resolve redundant relative paths" to handle just this case as well in the fileLoader if we don't want to override this function.

It may also be more appropriate to add this check to _CleanUrl as well

Reply:
Putting it in _CleanUrl seems right to me.

Reply:
Agree

Reply:
After talking with @sebavan, we decided that doing a simple if check for the "./" before checking the store would be the best approach